starter/source/materials/mat/write_matparam.F

```diff
@@ -97,9 +97,13 @@ SUBROUTINE WRITE_MATPARAM(MAT_ELEM,LEN)
 c     write material title
 c
       DO IMAT = 1,NUMMAT      
-        DO I=1,NCHARTITLE
-          NAME(I) = ICHAR(MAT_ELEM%MAT_PARAM(IMAT)%TITLE(I:I))
-        END DO
+        IF(IMAT < NUMMAT) THEN
+          DO I=1,NCHARTITLE
+            NAME(I) = ICHAR(MAT_ELEM%MAT_PARAM(IMAT)%TITLE(I:I))
```

Comment:
@MaciekWronski , 
MAT_PARAM(NUMMAT)%TITLE seems never initialized. The last material seems special (a lot of loops goes from 1 to NUMMAT-1). Is this change ok?